state/memcached/memcached.go

```diff
@@ -133,8 +136,15 @@ func (m *Memcached) parseTTL(req *state.SetRequest) (*int32, error) {
 		if err != nil {
 			return nil, err
 		}
+
 		parsedInt := int32(parsedVal)
 
+		// If ttl is more than 30 days, convert it to unix timestamp.
+		// https://github.com/memcached/memcached/wiki/Commands#standard-protocol
+		if parsedInt > 60*60*24*30 {
```

Comment:
```suggestion
		if parsedInt > (60*60*24*30)-1 {
```

Does this need a -1?